modules/appeals_api/.bruno_appeals_api/higher-level-reviews v0/Create HLR.bru

```diff
@@ -0,0 +1,148 @@
+}
+
+auth:bearer {
+  token: {{bearer_token}}
+}
+
+body:json {
+  {
+    "data": {
+      "type": "higherLevelReview",
+      "attributes": {
+        "informalConference": true,
+        "benefitType": "compensation",
+        "veteran": {
+          "ssn": "{{veteran_ssn}}",
+          "icn": "{{veteran_icn}}",
+          "firstName": "{{veteran_first_name}}",
+          "lastName": "{{veteran_last_name}}",
+          "birthDate": "{{veteran_birth_date}}",
+          "address": {
+            "addressLine1": "123 Somestreet",
+            "city": "Sometown",
+            "countryCodeIso3": "USA",
+            "zipCode5": "66002"
+          },
+          "phone": {
+            "countryCode": "1",
+            "areaCode": "555",
+            "phoneNumber": "8001111"
+          },
+          "email": "josie@example.com",
+          "timezone": "America/Chicago",
+          "homeless": false
+        },
+        "socOptIn": true,
+        "informalConferenceContact": "veteran",
+        "informalConferenceTime": "800-1200 ET",
+        "informalConferenceRep": {
+          "firstName": "Helen",
+          "lastName": "Holly",
+          "phone": {
+            "countryCode": "6",
+            "areaCode": "555",
+            "phoneNumber": "8001111"
+          }
+        }
+      }
+    },
+    "included": [
+      {
+        "type": "appealableIssue",
+        "attributes": {
+          "issue": "tinnitus",
+          "decisionDate": "1900-01-01",
+          "decisionIssueId": 1,
+          "ratingIssueReferenceId": "2",
+          "ratingDecisionReferenceId": "3",
+          "socDate": "2020-08-01"
+        }
+      },
+      {
+        "type": "appealableIssue",
+        "attributes": {
+          "issue": "left knee",
+          "decisionDate": "1900-01-02",
+          "decisionIssueId": 4,
+          "ratingIssueReferenceId": "5"
+        }
+      },
+      {
+        "type": "appealableIssue",
+        "attributes": {
+          "issue": "right knee",
+          "decisionDate": "1900-01-03",
+          "ratingIssueReferenceId": "6",
+          "ratingDecisionReferenceId": "7"
+        }
+      },
+      {
+        "type": "appealableIssue",
+        "attributes": {
+          "issue": "PTSD",
+          "decisionDate": "1900-01-04",
+          "decisionIssueId": 8,
+          "ratingDecisionReferenceId": "9"
+        }
+      },
+      {
+        "type": "appealableIssue",
+        "attributes": {
+          "issue": "Traumatic Brain Injury",
+          "decisionDate": "1900-01-05",
+          "decisionIssueId": 10
+        }
+      },
+      {
+        "type": "appealableIssue",
+        "attributes": {
+          "issue": "right shoulder",
+          "decisionDate": "1900-01-06"
+        }
+      }
+    ]
+  }
+}
+
+vars:pre-request {
+  oauth_scope: system/HigherLevelReviews.write
+}
+
+assert {
+  res.status: eq 201
+}
+
+script:pre-request {
+  const { setOauthToken } = require('./helpers.js');
+  await setOauthToken();
```

Comment:
Here's an example of using the oauth-token-fetching helper I added. It uses the `vars:pre-request` setting for `oauth_scope` that's configured just above this line, and fetches a token with that scope.